### PR TITLE
Fix mu convert enum to allow converting from a "from" enum to 2 different "to" enums

### PR DESCRIPTION
**Pull request by dcristoloveanu**

### CODE REVIEW

**Review by mattdurak — APPROVED** [review]
:shipit:

File: tests/mu_convert_enum_test.c [hunk 1]
```diff
@@ -37,6 +44,12 @@ MU_DEFINE_CONVERT_ENUM(TEST_CONVERT_FROM_ENUM, TEST_CONVERT_TO_ENUM,
     test_from_b, test_to_b,
     test_from_c, test_to_b);
 
+// This has to compile :-) to prove that we can map the same from enum to 2 different to enums
+MU_DEFINE_CONVERT_ENUM(TEST_CONVERT_FROM_ENUM, TEST_CONVERT_TO_ANOTHER_ENUM,
```
Comment (anporumb) [inline tests/mu_convert_enum_test.c:48]: >TEST_CONVERT_TO_ANOTHER_ENUM [](http://example.com/codeflow?start=47&length=28)

No tests to prove that the conversion doesn' produce some wonky result? TEST_CONVERT_TO_ENUM has C tests.

Comment (dcristoloveanu) [reply tests/mu_convert_enum_test.c:48]: Nothing that duplicating the current tests cannot fix. Done.

**Review by anporumb — APPROVED** [review]
:shipit:

**Pull request merged**